packages/core/src/sharedWishlists/redux/selectors.js

```diff
@@ -0,0 +1,151 @@
+ * import { getSharedWishlistError } from '@farfetch/blackout-redux';
+ *
+ * const mapStateToProps = state => ({
+ *    error: getSharedWishlistError(state)
+ * });
+ */
+export const getSharedWishlistError = state =>
+  fromSharedWishlistReducer.getError(state.sharedWishlist);
+
+/**
+ * Retrieves the loading status of the shared wishlist.
+ *
+ * @function
+ * @memberof module:sharedWishlists/selectors
+ *
+ * @param {object} state - Application state.
+ *
+ * @returns {boolean} Loading status of the shared wishlist.
+ *
+ * @example
+ * import { getSharedWishlistLoading } from '@farfetch/blackout-redux';
+ *
+ * const mapStateToProps = state => ({
+ *    isLoading: getSharedWishlistLoading(state)
+ * });
+ */
+export const getSharedWishlistLoading = state =>
+  fromSharedWishlistReducer.getIsLoading(state.sharedWishlist);
+
+/**
+ * Retrieves a specific shared wishlist by its id, with all properties populated(ie,
+ * the product).
+ *
+ * @function
+ * @memberof module:sharedWishlists/selectors
+ *
+ * @param {object} state - Application state.
+ * @param {string} sharedWishlistId - Numeric identifier of the shared wishlist.
+ *
+ * @returns {object} - Shared wishlist entity for the given id.
+ *
+ * @example
+ * import { getSharedWishlist } from '@farfetch/blackout-redux';
+ *
+ * const mapStateToProps = (state, { sharedWishlists: { id } }) => ({
+ *    sharedWishlist: getSharedWishlist(state, id)
+ * });
+ */
+export const getSharedWishlist = (state, sharedWishlistId) => {
+  return getEntity(state, 'sharedWishlists', sharedWishlistId);
+};
+
+/**
+ * Retrieves a specific shared wishlist item by its id, with all properties
+ * populated (ie, the product).
+ *
+ * @function
+ * @memberof module:sharedWishlists/selectors
+ *
+ * @param {object} state - Application state.
+ * @param {number} sharedWishlistItemId - Numeric identifier of the shared wishlist item.
+ *
+ * @returns {object} - Shared wishlist item entity for the given id.
+ *
+ * @example
+ * import { getSharedWishlistItem } from '@farfetch/blackout-redux';
+ *
+ * const mapStateToProps = (state, { sharedWishlistItem: { id } }) => ({
+ *    sharedWishlistItem: getSharedWishlistItem(state, id)
+ * });
+ */
+export const getSharedWishlistItem = (state, sharedWishlistItemId) => {
+  const sharedWishlistItem = getEntity(
+    state,
+    'sharedWishlistItems',
+    sharedWishlistItemId,
+  );
+
+  if (!sharedWishlistItem) {
+    return;
+  }
+
+  const product = getProduct(state, sharedWishlistItem.product);
+
+  return { ...sharedWishlistItem, product };
+};
+
+/**
+ * Retrieves shared wishlist items for a specficit wishlist.
+ *
+ * @function
+ * @memberof module:sharedWishlists/selectors
+ *
+ * @param {object} state - Application state.
+ * @param {number} sharedWishlistId - Numeric identifier of the shared wishlist.
+ *
+ * @returns {object} - Shared wishlist items entity for the given id.
+ *
+ * @example
+ * import { getSharedWishlistItem } from '@farfetch/blackout-redux';
```

Comment:
This is using `@farfetch/blackout-redux` which does not exist in v1. Please, review any references to these v2 packages and correct them.

Reply:
Updated!